<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagCleanerWidget</class>
 <widget class="QWidget" name="TagCleanerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>タグクリーナー</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelPrompt">
     <property name="text">
      <string>タグを入力してください（カンマ区切り）：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditPrompt">
     <property name="placeholderText">
      <string>例：1boy, 1girl, 青髪, 赤目</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFormat">
       <property name="text">
        <string>フォーマット：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFormat"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConvert">
       <property name="text">
        <string>変換</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelResult">
     <property name="text">
      <string>変換後のタグ：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
